### PR TITLE
Typo fix and added colons before code

### PR DESCRIPTION
**Pull request by ZviBaratz**
Minor suggestions.

### CODE REVIEW

**Comment by agramfort** [comment]
@ZviBaratz did you check if we are consistent with this in the tutorials? I would say it's better to be consistent and "wrong" with the missing :, than inconsistent ... so if we vote for the : we should change our "standard" in all tutorials

thoughts @drammock ?

thx @ZviBaratz

**Comment by ZviBaratz** [comment]
@agramfort right, my bad – it seems most tutorials alternate between periods and nothing. I saw the colon in the first part ([line 30](https://github.com/mne-tools/mne-python/blob/0f01b8dfa54ef6ae879b737d795c11574fc01517/examples/simulation/simulate_evoked_data.py#L30)), which personally looks better to me, and assumed the rest to be missing. I think in terms of this PR, it's probably best to just fix the typo and conform by removing the colon in the first part. 

If, however, you decide to add colons in phrases introducing code blocks, I'd be happy to help with that.

File: examples/simulation/simulate_evoked_data.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
           for ln in label_names]
 
 # %%
-# Generate source time courses from 2 dipoles and the correspond evoked data
+# Generate source time courses from 2 dipoles and the corresponding evoked data
```
Comment (cbrnr) [inline examples/simulation/simulate_evoked_data.py:52]: This is 80 chars - can you reformulate so that this comments fits in one line?

**Comment by larsoner** [comment]
Thanks @ZviBaratz !

**Pull request merged**